### DISCUSSION TITLE
가상패드의 영역을 지정할 수 있나요?

### DISCUSSION THREAD

**Discussion by JaeHyeon774** [naverz/zepeto-studio-kor#673] [5. Q&A]
![KakaoTalk_20220428_115216457](https://user-images.githubusercontent.com/55771437/165667328-272f91a2-a1e1-4069-b854-5f1f466a8755.jpg)
플레이어를 이동 시키는 패드가 화면 중간까지 와서 다른 버튼을 누르거나 할 때 이동하는 문제가 있습니다. 컨트롤 버튼을 하단에 고정하거나 잠시 비활성화 할 수 있는 방법이 있을까요?

**Comment by minjujuu** [comment]
ZepetoPlayers에서 사용하고 계신 Touchpad의 프리팹을 다음과 같이 수정해주시면 
화면을 차지하는 패드의 크기를 조절하실 수 있습니다!

![image](https://user-images.githubusercontent.com/79794414/165678487-d4e0a9f9-ccbc-476d-b0c3-3108dfc2744e.png)

![image](https://user-images.githubusercontent.com/79794414/165678540-dd9824cc-9014-4555-9385-2a42b622964d.png)

**Reply by JaeHyeon774** [reply]
답변 감사합니다. 그런데 수정을 해도 저장이 되지 않네요.
타입스크립트로 수정을해 야하는 건가요?

**Reply by x4cf** [reply]
리소스를 복사해서 Assets폴더 안에 넣은다음 복제한 리소스를 수정해야해요
Packages 폴더안의 리소스는 수정 불가능입니다 :D

**Reply by JaeHyeon774** [reply]
감사합니다!!! 덕분에 잘 됩니다!!!ㅎㅎㅎㅎ
